lib/active_record/pt_osc_migration.rb

```diff
@@ -263,5 +265,9 @@ def get_from_config(flag, options = {})
         flag
       end
     end
+
+    def execute(sql)
+      raise ActiveRecord::UnsupportedMigrationError.new("Raw `execute` isn't supported by the pt-osc gem.\n")
```

Comment:
The exception message shouldn't have a newline; that's up to the exception handler to add if needed.